docs/source/reference/math.rst

```diff
@@ -49,6 +49,7 @@ Rounding
 
    cupy.around
    cupy.round_
+   cupy.round
```

Comment:
Conventionally, functions undocumented (ie, not listed in the routines list) in NumPy are left undocumented in CuPy.

Reply:
@grlee77 Could you address this comment?

Reply:
I have removed it now.